tst/com/amazon/ion/benchmark/optionsCombinations.ion

```diff
@@ -1 +1 @@
-[" read --api dom --format ion_binary --results-format ion ", " read --api streaming --format ion_binary --results-format ion ", " write --api streaming --format ion_binary --results-format ion ", " read --api streaming --format json --results-format ion ", " read --api dom --format json --results-format ion ", " write --api streaming --format json --results-format ion "]
\ No newline at end of file
+[" read --api dom --format ion_binary --results-format ion --iterations 20 ", " read --api streaming --format ion_binary --results-format ion --iterations 20 ", " write --api streaming --format ion_binary --results-format ion --iterations 20 ", " read --api streaming --format json --results-format ion --iterations 20 ", " read --api dom --format json --results-format ion --iterations 20 ", " write --api streaming --format json --results-format ion --iterations 20 "]
```

Comment:
Is the increased number of iterations necessary for this technique to work? Do our Github Actions workflows run for that many iterations?